main.R

```diff
@@ -139,9 +139,9 @@ techexposure_data
 
 # input and output directories -------------------------------------------------
 
-input_dir <- "./inputs"
-output_dir <- "./outputs"
-data_dir <- "./data"
+input_dir <- "../inputs"
+output_dir <- "../outputs"
+data_dir <- "../data"
 
```

Comment:
The original was correct. The expectation for file I/O on an RStudio run is that `inputs`, `data` and `outputs` directories are nested within the repo itself. This is critical to ensure that the containerization functions as expected, see:

https://github.com/RMI-PACTA/workflow.pacta.dashboard/blob/320b90a76677bcc372650c356aa3c7cccc0d3aad/docker-compose.yml#L8
https://github.com/RMI-PACTA/workflow.pacta.dashboard/blob/320b90a76677bcc372650c356aa3c7cccc0d3aad/docker-compose.yml#L12
https://github.com/RMI-PACTA/workflow.pacta.dashboard/blob/320b90a76677bcc372650c356aa3c7cccc0d3aad/docker-compose.yml#L16 

Although, I realize now that we should probably add those directories to the `.gitignore` as well.

Dealt with in https://github.com/RMI-PACTA/workflow.pacta.dashboard/pull/16

Reply:
ok, now it makes sense. I didn't see them in .gitignore that is why I was confused about the structure.

Reply:
I just added them!! No problem :)